ui/app/models/auth-method.js

```diff
@@ -5,13 +5,13 @@
 
 import Model, { belongsTo, hasMany, attr } from '@ember-data/model';
 import { alias } from '@ember/object/computed'; // eslint-disable-line
-import { computed } from '@ember/object'; // eslint-disable-line
+import { action, computed } from '@ember/object'; // eslint-disable-line
```

Comment:
Can we remove `computed` since they should have all be converted to getters? It looks like `alias` can also be deleted, along with both eslint disable comments

Reply:
Yeah, not sure why my linter didn't catch this

Reply:
Oh, because eslint is disabled on that line 😂